__tests__/app.test.js

```diff
@@ -39,6 +39,53 @@ describe("/api/topics", () => {
   });
 });
 
+describe("/api/articles", () => {
+  test("GET 200: Respond with an articles array of article of objects containing 8 values", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article) => {
+          expect(Object.keys(article).length).toBe(8);
+        });
+      });
+  });
+
+  test("GET 200: Respond with an articles array of article objects with following properties: author, title, article_id, topic, created_at,votes, article_img_url, comment_count in correct format", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((article, index) => {
```

Comment:
When iterate through an array using a forEach in a test, you should always do a length check first. If the length is 0, then it won't run the .forEach, meaning it will skip all of the expect statements, and therefore will auto-pass!